samples/syrup

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+ROOT_DIR="$( dirname "${BASH_SOURCE[0]}" )/.."
+BUILD_DIR=.build/debug
+ORIGINAL_DIRECTORY=$(pwd)
+
+cd $ROOT_DIR
+ln -s $(pwd)/Templates $BUILD_DIR/Templates
```

Comment:
IIRC the CLI language option we specify can actually be a path to a template folder whereever. No need to symlink.

Reply:
I originally tried to do it with paths to the template folder but `syrup-gradle` [currently hardcodes the language](https://github.com/Shopify/syrup-gradle/blob/master/src/main/java/com/shopify/syrup/tasks/GenerateModelsTask.kt#L24) for the template option which Kotlin currently uses for generation. I could look through syrup arguments for the language then replace it with the folder location.